kubernetes/samples/scripts/domain-lifecycle/setupPortForwarding.sh

```diff
@@ -0,0 +1,181 @@
+# !/bin/sh
+# Copyright (c) 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/helper.sh
+if [ "${debug}" == "true" ]; then set -x; fi;
+set -eu
+
+function usage() {
+
+  cat << EOF
+
+  This script sets up the 'kubectl port-forward' for a WebLogic domain to allow access to the
+  WebLogic Administration Console. The 'kubectl port-forward' pattern is often more secure than
+  exposing the console to the public Internet. The script creates a pod in the domain namespace
+  using the image containing 'socat' binary and it forwards the request to the WebLogic
+  Admistration Server pod. The script derives the address and the adminstration port of the
+  WebLogic Admistration Server using the domain topology. The port can also be provided on the
+  command line using '-r' option.
```

Comment:
Usage could use more detail:
- Make it more clear that the console requests redirect from the port-forward, through a 'socat pod' on to the admin server. (something like "it forwards the request" --> "the script runs a port-forward command which forwards console requests to the socat pod which in turn forwards requests to the admin server pod'
- Explain that the script runs a port-forward command in foreground after launching the pod and a that a ^C not only terminates the port-forward command but also implicitly tears down the pod.